Gemfile.lock

```diff
@@ -489,7 +489,7 @@ GEM
     ruby-progressbar (1.7.1)
     rubyntlm (0.4.0)
     rubypants (0.2.0)
-    rugged (0.21.2)
+    rugged (0.21.4)
```

Comment:
Sure. The only difference is the bundled libgit2.

https://github.com/libgit2/rugged/blob/master/CHANGELOG.md#rugged-0214-january-18-2015

libgit2 changelog is a bit messy, but the 0.21.5 seems to be a maintenance version:
https://github.com/libgit2/libgit2/releases
So I guess 0.21.4 was too.